### ISSUE TITLE
Temperature and Humidity showing 0

### ISSUE THREAD

**Issue by r1mad** [fantasytu/homebridge-mijia-thermometer#8]
**Describe Your Problem:**
I've managed to install the plugin and config json file. After accessories were added they are showing 0 degrees both in homebridge and homekit. Did you notice that issue until now and is there any solution?
Thanks!

**Comment by Searge** [comment]
The same:

Autoconfigured
![MijiaThermometer](https://user-images.githubusercontent.com/2125311/174438678-740a21a6-c42c-4610-a290-e9eb406aaa47.png)

Added to the Home, no changes...

**Comment by atomicvindaloo** [comment]
Did you ever get anywhere with this? I have the same problem; Plugin installed and configured, but temperature and humidity showing zero.

**Comment by emreisik95** [comment]
same issue here. please keep me in the loop.

**Comment by ZeljkoGacic** [comment]
Hi, I have same issue, anyone manage to overcome it?

**Comment by maleeeeeee** [comment]
Same issue. Is there any solution ?

**Comment by michalidesko** [comment]
0 too :/

**Comment by KHS-source** [comment]
Use [Mi Hygrothermograph](https://github.com/hannseman/homebridge-mi-hygrothermograph) plugin instead. It works like a charm.

**Comment by BenoitDeguine** [comment]
Any alternative in 2023?

**Comment by searge** [comment]
> Any alternative in 2023?

Yeah, I've switched to this: https://github.com/fantasytu/homebridge-mijia-thermometer#readme

**Comment by BenoitDeguine** [comment]
Are you sure of the link? It's the same as the current project

**Comment by searge** [comment]
> Are you sure of the link? It's the same as the current project

Sorry, yes.
It's `homebridge-mijia-thermometer-fix v1.7.6`

![image](https://github.com/fantasytu/homebridge-mijia-thermometer/assets/2125311/9ef2fcaa-82d5-4658-9783-34c81a58d4e0)
